mne/io/edf/edf.py

```diff
@@ -324,46 +309,7 @@ def _parse_tal_channel(tal_channel_data):
 
 def _get_edf_info(fname, stim_channel, annot, annotmap, tal_channel,
                   eog, misc, preload):
-    """Extracts all the information from the EDF+,BDF file.
-
-    Parameters
-    ----------
-    fname : str
-        Raw EDF+,BDF file to be read.
-    stim_channel : str | int | None
-        The channel name or channel index (starting at 0).
-        -1 corresponds to the last channel.
-        If None, there will be no stim channel added.
-    annot : str | None
-        Path to annotation file.
-        If None, no derived stim channel will be added (for files requiring
-        annotation file to interpret stim channel).
-    annotmap : str | None
-        Path to annotation map file containing mapping from label to trigger.
-        Must be specified if annot is not None.
-    tal_channel : int | None
-        The channel index (starting at 0).
-        Index of the channel containing EDF+ annotations.
-        -1 corresponds to the last channel.
-        If None, the annotation channel is not used.
-        Note: this is overruled by the annotation file if specified.
-    eog : list of str | None
-        Names of channels that should be designated EOG channels. Names should
-        correspond to the electrodes in the edf file. Default is None.
-    misc : list of str | None
-        Names of channels that should be designated MISC channels. Names
-        should correspond to the electrodes in the edf file. Default is None.
-    preload : bool
-        If True, all data are loaded at initialization.
-        If False, data are not read until save.
-
-    Returns
-    -------
-    info : instance of Info
-        The measurement info.
-    edf_info : dict
-        A dict containing all the EDF+,BDF  specific parameters.
-    """
+    """Extracts all the information from the EDF+,BDF file"""
```

Comment:
FYI the place I noticed it especially here was in `_read_segment`, where I would have had to update three places to add another argument whose definition is already clear from the publicly visible function in `_BaseRaw`, and it didn't really add much to the code readability to have the comments. (I actually found that it decreased readability a bit.) In some other places like forward calculation what @mainakjas was doing -- adding comments to clarify what everything was -- is more useful and thus justified. My 2c anyway.